app/assets/stylesheets/retros.scss

```diff
@@ -49,5 +49,25 @@ html {
         }
       }
     }
+
+    #Boards {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+
+      #discuss {
+        border-width: 1px;
+        width: 200px;
+        height: 300px;
+        margin-top: 10%;
```

Comment:
Not specified in the story, but if you also need the section to be vertically centered I recommend doing that using flex-box instead of using the margin-top property in this element.